xdsl/interactive/app.tcss

```diff
@@ -21,12 +28,12 @@
 }
 
 
-# Horizontal( SelectionList, ScrollableContainer(Label) )
+# Horizontal( SelectionList, Horizontal(Button), ScrollableContainer(Label) )
 #selected_passes_and_list_horizontal{
     height: 100%;
     layout: grid;
-    grid-size: 2 1;
-    grid-columns: 50% 50%;
+    grid-size: 3 1;
+    grid-columns: 50% 10% 40%;
```

Comment:
Would it make sense to keep the grid be 50-50, and then the buttons that act on either the pass pipeline builder or pass pipeline representaiton be within their section of the grid? the 10% seems a bit arbitrary, since the buttons are of a fixed dimension.

Reply:
Could you please give this a go? There should be a way to say that the button has whatever size it wants.